api/tempo/v1alpha1/tempostack_types.go

```diff
@@ -177,19 +177,32 @@ type MetricsConfigSpec struct {
 // TracingConfigSpec defines a tracing config including endpoints and sampling.
 type TracingConfigSpec struct {
 	// SamplingFraction defines the sampling ratio. Valid values are 0 to 1.
+	// The SamplingFraction has to be defined to enable tracing.
 	//
 	// +optional
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Sampling Fraction"
 	SamplingFraction string `json:"sampling_fraction,omitempty"`
 
 	// JaegerAgentEndpoint defines the jaeger endpoint data gets send to.
+	// Deprecated: in favor of OTLPHttp.
 	//
 	// +optional
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:default:="localhost:6831"
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Jaeger-Agent-Endpoint"
 	JaegerAgentEndpoint string `json:"jaeger_agent_endpoint,omitempty"`
+
+	// OTLPHttp defines the OTLP/http endpoint data gets send to.
+	// For example, "http://localhost:4320".
+	// The default OTLP/http port 4318 collides with the distributor ports, therefore it is recommended to use a different port
+	// on the sidecar injected to the Tempo (e.g. 4320).
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default:="http://localhost:4320"
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="OTLP-HTTP-Endpoint"
+	OTLPHttp string `json:"otlp_http,omitempty"`
```

Comment:
```suggestion
	OTLPHttpEndpoint string `json:"otlp_http_endpoint,omitempty"`
```

I would add the `_endpoint` suffix to the field name in the CRD, to be in sync with the OTEL terminology (e.g. the env var is called `OTEL_EXPORTER_OTLP_ENDPOINT` or the setting `exporters.otlphttp.endpoint: https://example.com:4318` in the collector configuration)